geode-core/src/main/java/org/apache/geode/internal/admin/SSLConfig.java

```diff
@@ -41,6 +41,7 @@
   private String truststoreType = KeyStore.getDefaultType();
   private String alias = null;
   private SecurableCommunicationChannel securableCommunicationChannel = null;
+  private boolean skipSslVerification;
```

Comment:
I am not sure if this flag belongs to SSLConfig. Only ConnectCommand uses this concept, we can just keep it in there and pass the flag down to httpConnect()